### PR TITLE
Add some useful aliases and functions

### PR DESCRIPTION
**Pull request by jusdepatate**
Yet Another Pull Request:tm:

I've added some things like 
- cdls (cd and ls)
- mkdirr (mkdir, cd and ls)
- cdh (cd ~) 
- lisah (ls -lisah)

### CODE REVIEW

File: aliases/available/general.aliases.bash [hunk 1]
```diff
@@ -15,9 +15,14 @@ alias la='ls -AF'       # Compact view, show hidden
 alias ll='ls -al'
 alias l='ls -a'
 alias l1='ls -1'
+alias lisah='ls -lisah --color=auto'
 
 alias _="sudo"
 
+function mkdirr() {
```
Comment (nwinkler) [inline aliases/available/general.aliases.bash]: This shouldn't be in an `alias` file - please move this to the `base` plugin.

File: aliases/available/general.aliases.bash [hunk 2]
```diff
@@ -57,11 +62,16 @@ alias ipy='ipython'
 alias piano='pianobar'
 
 alias ..='cd ..'         # Go up one directory
+alias cdh='cd ~/'  # Go in home
```
Comment (nwinkler) [inline aliases/available/general.aliases.bash]: I don't think this is necessary. You can simply run `cd` (without any parameters) and it should take you to your home directory.

File: aliases/available/general.aliases.bash [hunk 3]
```diff
@@ -57,11 +62,16 @@ alias ipy='ipython'
...
 alias cd..='cd ..'       # Common misspelling for going up one directory
 alias ...='cd ../..'     # Go up two directories
 alias ....='cd ../../..' # Go up three directories
 alias -- -='cd -'        # Go back
 
+function cdls() {
```
Comment (nwinkler) [inline aliases/available/general.aliases.bash]: Same here, please move this to the `base` plugin.

File: aliases/available/general.aliases.bash (hunk 1 above)
Comment (jusdepatate) [reply aliases/available/general.aliases.bash]: https://github.com/jusdepatate/bash-it/commit/962f3093a2fa8c364e72d8e970ce7e349e8caa54

File: aliases/available/general.aliases.bash (hunk 2 above)
Comment (jusdepatate) [reply aliases/available/general.aliases.bash]: https://github.com/jusdepatate/bash-it/commit/cc91898eefb92c193880349bbbd548eda6efb784

File: aliases/available/general.aliases.bash (hunk 3 above)
Comment (jusdepatate) [reply aliases/available/general.aliases.bash]: [jusdepatate@962f309](https://github.com/jusdepatate/bash-it/commit/962f3093a2fa8c364e72d8e970ce7e349e8caa54)

File: plugins/available/base.plugin.bash [hunk 4]
```diff
@@ -1,6 +1,14 @@
 cite about-plugin
 about-plugin 'miscellaneous tools'
 
+
+function mkdirr() {
+    mkdir $1 && cd $1 && ls
+}
+function cdls() {
```
Comment (nwinkler) [inline plugins/available/base.plugin.bash]: Can you please add an empty line between these two functions?

File: plugins/available/base.plugin.bash [hunk 5]
```diff
@@ -1,6 +1,14 @@
 cite about-plugin
 about-plugin 'miscellaneous tools'
 
+
+function mkdirr() {
```
Comment (nwinkler) [inline plugins/available/base.plugin.bash]: Can you please add the missing function headers to both of your new functions? Please take a look at the `down4me` function a couple of lines down.

You'd probably need to add the `about`, `param`, `example` and `group` headers to both functions.

**Comment by NoahGorny** [comment]
Hey, seems like this PR has gone stale. So I am closing it. Thanks for contributing!

**Pull request closed without merging**